snap/info.go

```diff
@@ -88,6 +88,8 @@ type Info struct {
 	Plugs            map[string]*PlugInfo
 	Slots            map[string]*SlotInfo
 
+	Prices map[string]float64
```

Comment:
This should be in SideInfo rather than here. SideInfo contains data that is not in the snap itself, and is persisted in the state.